Components/Maps/Maps.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import { StyleSheet, View, Text, Dimensions } from 'react-native';
+import MapView, { PROVIDER_GOOGLE } from 'react-native-maps';
+import { Agenda } from 'react-native-calendars'
+import { Marker } from 'react-native-maps';
+
+
+export const Maps = ({ route }) => {
+  const { destination, startDate, endDate, name } = route.params
+  console.log('lat/long: ', destination);
+  console.log('dates: ', startDate, endDate);
+
+  const [ marker, setMarker ] = useState({});
+
+  return (
+    <View style={styles.container}>
+      <MapView style={styles.mapStyle}
+        provider={PROVIDER_GOOGLE}
+        region={{
+          latitude: parseFloat(destination.lat),
+          longitude: parseFloat(destination.long),
+          latitudeDelta: 0.5,
+          longitudeDelta: 0.55
+        }}
+        showsUserLocation
+      >
+      <Marker 
+        coordinate={{
+          latitude: parseFloat(destination.lat),
+          longitude: parseFloat(destination.long),
+          latitudeDelta: 0.4,
+          longitudeDelta: 0.41
+        }}
+      />
+      </MapView>
+      <Agenda style={styles.agenda}
+        items={{
+          [startDate]: { name: name, location: destination.location },
+          [endDate]: { name: name, location: destination.location }
+        }}
+        renderEmptyDate={() => { return <View /> }}
+      />
+    </View>
+  )
+}
+
+export default Maps;
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    // justifyContent: 'center',
+  },
+  mapStyle: {
+    width: Dimensions.get('window').width,
+    height: Dimensions.get('window').height/3,
+  },
+  agenda: {
+    width: Dimensions.get('window').width,
+  }
+})
```

Comment:
So good to see this up and running! I appreciated your work on this. I'm excited to get the agenda view up and running. If I remember correctly, it seems as though we need to add some logic to show the agenda, wether or not the user has an event that day or not. I'd be happy to dig into it!

Reply:
Everything is looking really good so far, great job on this! So awesome to see this all coming together, really enjoying viewing it on mobile. Keep crushing it and please let us know if we can assist in any way.

Reply:
This looks awesome, Colin! Like Zac said, let us know if we can help you with anything. It's really coming together!